firefox-ios/firefox-ios-tests/Tests/XCUITests/MicrosurveyTests.swift

```diff
@@ -0,0 +1,66 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+final class MicrosurveyTests: BaseTestCase {
+    override func setUp() {
+        launchArguments = [
+            LaunchArguments.SkipIntro,
+            LaunchArguments.ResetMicrosurveyExpirationCount
+        ]
+        super.setUp()
+    }
+
+    func testShowMicrosurveyPromptFromHomepageTrigger() {
```

Comment:
I am wondering if these tests are matching scenarios with testrail test cases. If not we could add the missing links in the future.

Reply:
yeah, that's why I commented in our slack channel. Currently draft version of tests are [here](https://testrail.stage.mozaws.net/index.php?/suites/view/53811&group_by=cases:section_id&group_order=asc&display_deleted_cases=0) they will be moved to the FullFunctional test suite when ready, and then we can match them to add the link.
If you can take a look and see if we should modify something while being in draft mode, that would help when they are ready. Thanks!

Reply:
We will match them, no worries about that